src/Actions/ManagesWorkflow.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
```

Comment:
All the functions should have appropriate docblocks. Please check the other classes in the repo and see how it is done.